test/OpenTelemetry.Collector.Dependencies.Tests/BasicTests.cs

```diff
@@ -77,8 +77,8 @@ public async Task HttpDependenciesCollectorInjectsHeadersAsync()
             parent.ActivityTraceFlags = ActivityTraceFlags.Recorded;
 
             using (new HttpClientCollector(tracer, new HttpClientCollectorOptions()))
-            using (var c = new HttpClient())
             {
+                using var c = new HttpClient();
```

Comment:
This case doesn't seem that improves readability at all. Same identation, just moved the declaration to the inner scope.